Package.swift

```diff
@@ -17,38 +17,24 @@ let package = Package(
         .package(name: "Autofill", url: "https://github.com/duckduckgo/duckduckgo-autofill.git", .exact("4.7.1")),
         .package(name: "GRDB", url: "https://github.com/duckduckgo/GRDB.swift.git", .exact("1.1.0")),
         .package(url: "https://github.com/duckduckgo/TrackerRadarKit", .exact("1.0.4")),
-        .package(name: "Punycode", url: "https://github.com/gumob/PunycodeSwift.git", .exact("2.1.0"))
+        .package(name: "Punycode", url: "https://github.com/gumob/PunycodeSwift.git", .exact("2.1.0")),
+        .package(url: "https://github.com/duckduckgo/content-scope-scripts", .branch("jkt/remote-values"))
```

Comment:
Just to be sure, is `jkt/remote-values` branch desired?

Reply:
At this time yes. It will be pinned to the version when that branch is merged as part of this project.